src/pages/Events/EventsList.tsx

```
@@ -569,11 +624,77 @@ export default function EventsList() {
   // ── Issue #2664: Optimistic UI for RSVP and Bookmark ─────────────
-    // ── onSuccess: subtle success toast ─────────────────────────────
+  const toggleRsvp = useMutation({
+    mutationFn: async ({ eventId, hasRsvpd }: { eventId: string; hasRsvpd: boolean }) => {
+      if (!user) throw new Error("Must be logged in");
+      if (eventId.startsWith("mock-")) {
+        return;
+      }
+
+      // Check if it's a remote/federated event in allEvents array
+      const targetEvent = allEvents.find((e) => e.id === eventId);
+      if (targetEvent && 'is_remote' in targetEvent && targetEvent.is_remote) {
```

ESLint

[error] Replace `'is_remote'` with `"is_remote"`

(prettier/prettier)